aidants_connect_habilitation/admin.py

```diff
@@ -1 +1,22 @@
-# Register your models here.
+from django.contrib.admin import ModelAdmin
+
+from aidants_connect.admin import admin_site, VisibleToAdminMetier
+from aidants_connect_habilitation.models import Issuer, OrganisationRequest
+
+
+class IssuerAdmin(VisibleToAdminMetier, ModelAdmin):
+    list_display = (
+        "email",
+        "last_name",
+        "first_name",
+        "phone",
+        "id",
+    )
+
+
+class OrganisationRequestAdmin(VisibleToAdminMetier, ModelAdmin):
+    raw_id_fields = ("issuer",)
+
+
+admin_site.register(Issuer, IssuerAdmin)
+admin_site.register(OrganisationRequest, OrganisationRequestAdmin)
```

Comment:
On peut utiliser [le décorateur `@register`](https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#the-register-decorator), maintenant qu'on est en 3.2 :partying_face:

Reply:
Suite à ton commentaire j'ai commencé à utiliser le décorateur et j'ai trouvé ça chic. Ensuite je me suis dit "hey mais il faudrait un feature flag…" du coup j'ai tout controle-zédé T_T